backend/db/sqliteschema/migrations.go

```diff
@@ -154,7 +165,7 @@ var migrations = []string{
 		time INTEGER NOT NULL CHECK (time > 0),
 		-- Abbreviated device ID that authored the change.
 		origin INTEGER NOT NULL CHECK (origin != 0),
-		PRIMARY KEY (permanode, change, time)
+		PRIMARY KEY (permanode, time, change, origin)
```

Comment:
What's origin?

Reply:
Its 8-byte prefix of the public key converted to uint64. It will change during the breaking change, so don't pay too much attention to that :)